src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -56,6 +59,9 @@
 
     /**
      * Test body.
+     * @todo: #120 move constants in this file to their tests because tests
```

Comment:
@westonized the puzzle has incorrect format. It should be `@todo #128:XXmin` where XX is number of minutes for a task. First line should be task title. Next lines are detailed task description